ShippoTesting/TrackTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Shippo;
+
+namespace ShippoTesting
+{
+    [TestFixture ()]
+    public class TrackTest : ShippoTest
+    {
+        [Test ()]
+        public void TestValidGetStatus ()
+        {
+            Shipment shipment = ShipmentTest.getDefaultObject ();
+            Track track = getAPIResource ().RetrieveTracking ("usps", shipment.ObjectId);
```

Comment:
Wait I just realized doesn't this take tracking number given by the carrier. Instead takes shipment ID from shippo. If so, how is this test working?